src/query_parser/v2/parser.y

```diff
@@ -464,6 +464,43 @@ expr(A) ::= modifier(B) COLON text_expr(C) . {
     }
 }
 
+expr(A) ::= modifier(B) COLON LP EMPTY_STRING RP . {
+  // printf("Empty text\n");
+  char *s = rm_strndup(B.s, B.len);
+  size_t slen = unescapen(s, B.len);
+
+  const FieldSpec *fs = IndexSpec_GetField(ctx->sctx->spec, s, slen);
+  if (!fs) {
+    // Non-existing field
+    reportSyntaxError(ctx->status, &B, "Syntax error: Field not found");
+    A = NULL;
+    rm_free(s);
```

Comment:
Today when such a situation is encountered (i.e., the field requested is not in the schema) we just don't return results, with no error thrown. Let's be consistent with that behavior.

Reply:
Although I 100% agree that we should throw an error..
This should be broken in `3.0` probably.